wpilibcExamples/src/main/cpp/examples/examples.json

```diff
@@ -610,7 +610,6 @@
     ],
     "foldername": "StateSpaceFlywheel",
     "gradlebase": "cpp",
-    "mainclass": "Main",
```

Comment:
Is the mainclass setting no longer required?

Reply:
In C++ it was never needed (the examples that had it were copy-pasta issues), and for Java it was made sure to be there but never used.

Reply:
Is it used by VS Code?